### ISSUE TITLE
dataloader SiameseDataset types 类别缺少最后一类 ？

### ISSUE THREAD

**Issue by se202242** [bubbliiiing/Siamese-pytorch#38]
utils.dataloader.SiameseDataset
self.types          = max(labels)
后续getitem 函数中 
c               = random.randint(0, self.types - 1)

label = self.types 类是否一直未被选中

更改为 
self.types          = max(labels) + 1 
程序仍跑通

**Comment by bubbliiiing** [comment]
randint是包括self.types - 1的嘞

**Comment by se202242** [comment]
但 labels 中 max 所对应类别不就一直未被取到吗？

**Comment by bubbliiiing** [comment]
self.types - 1就是最大对应的
